src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinstallment.phtml

```diff
@@ -14,11 +22,18 @@ $backends = $this->getInstallmentBackends();
                 <div class="logo-<?=$backend->_bankcode;?>"></div>
             </div>
             <div class="omise-banking-text-wrapper">
-                <span class="title"><?=$backend->_bankname;?></span><br/>
+                <span class="title"><?=$installmentText;?></span><br/>
                 <select name="payment[terms_<?=$backend->_id;?>]" id="payment_term_<?=$backend->_id;?>">
-                    <?php foreach ($backend->allowed_installment_terms as $termMonths) { ?>
-                        <option value="<?=$termMonths;?>"><?=$termMonths;?> <?=$this->__('months');?></option>
-                    <?php } ?>
+
+                    <?php foreach ($backend->allowed_installment_terms as $termMonths) {
```

Comment:
the same as above. PHP added designed to embed in html if/foreach/for statements, we should use them.